itou/jobs/models.py

```diff
@@ -7,6 +7,7 @@
 
 
 class Rome(models.Model):
+    updated_at = models.DateTimeField(auto_now=True)
```

Comment:
Il me semble que tu disais que les Rome peuvent changer de définition avec un code donné (un peu comme les Communes INSEE).
Du coup si on meut juste à jour l'objet en DB, on risque de modifier des anciennes JobDescriptions.

Est-ce qu'utiliser le mixin de Period (qui ajoute un start_date + end_date) pour préciser sur quelle période le Rome est valide ne permettrait pas d’éviter des soucis ?

Reply:
En effet, j'ai meme un diff de ce qui peut arriver.

En 3 ans:
- les ROME n'ont pas changé, on a les mêmes codes dans le meme nombre (532)
- certains libellés ont été modifiés: en gros on est passé de l'écriture "inclusive" à la double forme (Chef(fe) de gare => Chef / Cheffe de gare)
- les Appellation ont augmenté, on en a ~600 de plus
- certains labels modifiés de la meme manière
- a priori aucun supprimé, donc ce cas est super rare.

Ma logique est la suivante:
- Cas UPDATE: on se fiche un peu de l'évolution des labels des ROME et Appellation, tant mieux si ces derniers collent à la définition en cours coté Pole Emploi dans Les Emplois de l'Inclusion.
- Cas ADDITION: si un ROME ou une Appellation est ajoutée, on pourra du coup lier des fiches de poste à ces dernières. On a la date d'ajout du Rome ou de l'Appellation.
- Cas DELETION: si un ROME ou une Appellation est supprimée coté PE, on ne le supprime pas coté les Emplois. On sait juste que sa date de dernière modification va doucement devenir ancienne. En effet, supprimer le ROME ou l'Appellation entraînerait la suppression de toutes les fiches de poste associées, or il n'y  a pas de raison de faire ça.

Du coup je ne suis pas certain qu'on ait besoin de ce mixin de Period, à moins que tu ne penses qu'il soit utile d'enregistrer à quelle date exactement PE a cessé de nous envoyer le ROME ou l'Appellation en question ?

Reply:
Est-il important d'empêcher d'utiliser un ROME ou une Appellation supprimée pour des nouveaux postes ? 
Si non, on peut en effet s'en moquer et ce que tu proposes dans la PR suffit largement pour répondre au besoin (simpler is better)

Reply:
C'est un bon point, je vais demander au métier.